config/playground/helm/dev/dashboards.yaml

```diff
@@ -0,0 +1,203 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+opensearchHosts: "https://opensearch-cluster-master:9200"
+replicaCount: 1
+
+image:
+  repository: "opensearchproject/opensearch-dashboards"
+  # override image tag, which is .Chart.AppVersion by default
+  tag: ""
+  pullPolicy: "IfNotPresent"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+rbac:
+  create: true
+
+# A list of secrets and their paths to mount inside the pod
+# This is useful for mounting certificates for security and for mounting
+# the X-Pack license
+secretMounts: []
+#  - name: certs
+#    secretName: dashboard-certs
+#    path: /usr/share/dashboards/certs
+
+podAnnotations: {}
+
+extraEnvs: []
+#  - name: "NODE_OPTIONS"
+#    value: "--max-old-space-size=1800"
+
+envFrom: []
+
+extraVolumes: []
+  # - name: extras
+  #   emptyDir: {}
+
+extraVolumeMounts: []
+  # - name: extras
+  #   mountPath: /usr/share/extras
+  #   readOnly: true
+
+extraInitContainers: ""
+
+extraContainers: ""
+
+podSecurityContext: {}
+
+securityContext:
+  capabilities:
+    drop:
+      - ALL
+  # readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1000
+
+config: 
+  # Default OpenSearch Dashboards configuration from docker image of Dashboards
+  opensearch_dashboards.yml: |
+    opensearch_security.auth.anonymous_auth_enabled: true
+    opensearch.hosts: [https://localhost:9200]
+    opensearch.ssl.verificationMode: none
+    opensearch.username: kibanaserver
+    opensearch.password: kibanaserver
+    opensearch.requestHeadersWhitelist: [authorization, securitytenant]
+    
+    opensearch_security.multitenancy.enabled: true
+    opensearch_security.multitenancy.tenants.preferred: [Private, Global]
+    opensearch_security.readonly_mode.roles: [kibana_read_only]
+    # Use this setting if you are running opensearch-dashboards without https
+    opensearch_security.cookie.secure: false
+    server.host: '0.0.0.0'
+  #   server:
+  #     name: dashboards
+  #     host: 0.0.0.0
+
+  # Dashboards TLS Config (Ensure the cert files are present before enabling SSL
+      # ssl:
+      #   enabled: true
+      #   key: /usr/share/opensearch-dashboards/certs/dashboards-key.pem
+      #   certificate: /usr/share/opensearch-dashboards/certs/dashboards-crt.pem
+
+    # determines how dashboards will verify certificates (needs to be none for default opensearch certificates to work)
+    # opensearch:
+    #   ssl:
+    #     certificateAuthorities: /usr/share/opensearch-dashboards/certs/dashboards-root-ca.pem
+    #     if utilizing custom CA certs for connection to opensearch, provide the CA here
```

Comment:
Do we regard enabling TLS as a best practice?

Reply:
@Flyingliuhub Thoughts? I believe that it is but I would have to configure my own key, certificate, and CA.